### ISSUE TITLE
Incoming call ui is not getting removed on accepting the call on mi devices(Android 11) 

### ISSUE THREAD

**Issue by govi757** [react-native-webrtc/react-native-callkeep#484]
# Bug report

- I am using react-native-callkeep for displaying incoming call ui on receiving of firebase notification. We are able to display the ui.But on pressing accept button i am unable to hide the incoming call screen on some android 11 devices.

- Reproduced on:
 - [x] Android

## Description
We are able to display the ui.But on pressing accept button i am unable to hide the incoming call screen on some android 11 devices.
## Steps to Reproduce

`bgMessaging = async (message, ...props) => {
  
    RNCallKeep.setup(options);
    RNCallKeep.addEventListener('answerCall', async ({ callUUID }) => {
      RNCallKeep.endAllCalls();
    });
    RNCallKeep.displayIncomingCall("uuid", "Display", "Display", "number");
  }
  return Promise.resolve();
}
AppRegistry.registerHeadlessTask('RNFirebaseBackgroundMessage', () => bgMessaging);`

## Versions
    - Callkeep:^4.2.0
    - React Native:0.63.4
    - Android: 11
    - Phone model: Redmi note 9 

## Logs

```
Paste here
```

**Comment by smismailalam** [comment]
i am facing same issue

**Comment by HepaCoder** [comment]
Yeah I was facing the same issue too. Calling  `RNCallKeep.setCurrentCallActive(callUUID);` after the "answerCall" event solved the issue for me.

**Comment by azhararmar** [comment]
FOr me RNCallKeep.addEventListener('answerCall') never gets called

**Comment by fukemy** [comment]
any solution now?

**Comment by DinoZenka** [comment]
Same minor bug.  Anyone fixed that bug?

**Comment by fukemy** [comment]
remove this useless lib and use react-native-incoming-call. This lib only work for ios
